src/baldr/CMakeLists.txt

```diff
@@ -74,6 +71,12 @@ set(sources
     verbal_text_formatter_us_tx.cc
     verbal_text_formatter_factory.cc)
 
+set (sources_with_warnings
+    graphreader.cc
+    graphtile.cc
+    graphtileheader.cc
+    )
```

Comment:
We have the same convention for CMakeLists.txt as for C++ files where the parentheses closes on the same line as the last element

Reply:
Thank you I have changed this